### PR TITLE
Demo Sixt

### PR DESCRIPTION
**Pull request by repository-sync**
## PR Summary

🎉 Willkommen Sixt 🎉 

image

gif

## Checklist
- [ ] [Check our CI system](https://github-demo.ci.cloudbees.com/view/jonico-Hyrule/job/Jenkins%20Pipeline%20as%20code%20test/job/reading-time-app/) for feedback on your build
- [x] Reference at least one issue your pull request adresses
- [x] Ask `Hyrule/developers` for feedback
- [ ] Merge your pull request if your team gave :+1: and your change looks good in the [preview environment](https://jonico-reading-time-preview.herokuapp.com/)

### CODE REVIEW

**Comment by repository-sync** [comment]
@jonico: Can you have some 👀  on the changes
@infoCup2017: you too please

File: src/main/java/com/github/demo/service/BookService.java [hunk 1]
```diff
@@ -14,7 +14,6 @@
         books.add(new Book("Eric Ries","The Lean Startup: How Constant Innovation Creates Radically Successful Businesses", "lean.jpg"));
         books.add(new Book("Geoffrey A. Moore","Crossing the Chasm", "chasm.jpg"));
         //books.add(new Book("David Thomas","The Pragmatic Programmer: From Journeyman to Master", "pragmatic.jpg"));
-        books.add(new Book("Frederick P. Brooks Jr.", "The Mythical Man-Month: Essays on Software Engineering", "month.jpg"));
```
Comment (infoCup2017) [inline src/main/java/com/github/demo/service/BookService.java]: https://github.com/jonico/reading-time-app/blob/c456bcf068c10d19c311c6305c9b43915b7080f1/src/test/java/com/github/demo/service/BookServiceTest.java#L22

@repository-sync: ☝️ , this is the problematic line, please fix it

**Comment by infoCup2017** [comment]
@repository-sync: there is more work to do, see my inline changes

File: src/main/java/com/github/demo/service/BookService.java (hunk 1 above)
Comment (repository-sync) [reply src/main/java/com/github/demo/service/BookService.java]: @infoCup2017: better now?

**Pull request merged**